components/root_layout/root_layout.vue

```diff
@@ -84,8 +75,7 @@ export default {
     },
 
     /**
-     * The height of the header
-     * Possible units rem|px|%|em
+     * DEPRECATED: set the height of the inner element instead.
```

Comment:
When we will remove these `DEPRECATED` props?

Reply:
> When we will remove these `DEPRECATED` props? after checking it with the usage on web dev?

Yep exactly, after we check and fix any code still using them.